src/django_components/templatetags/component_tags.py

```diff
@@ -560,6 +575,88 @@ def _parse_tag_body(parser: Parser, end_tag: str, inline: bool) -> NodeList:
     return body
 
 
+def _fix_nested_tags(parser: Parser, token: Token) -> None:
+    block_token = token
```

Comment:
Is the intent here to just rename the variable? Can't this be done in the def right above?